homeassistant/brands/mazda.json

```diff
@@ -0,0 +1,5 @@
+{
+  "domain": "mazda",
```

Comment:
@bdr99 @bachya 

There are two things with this PR.

1) This PR adds a brand but does not provide multiple integrations. At this point, we only have brands that have multiple integrations or standards. This PR seems to violate/change that behavior. Was that discussed somewhere, or did we change that behavior anywhere?

2) There is a brands PR missing that adds this brand to the brands repository. Please make sure that PR is created before the next release.

IMHO both need to be resolved in order to keep this in, otherwise, we should revert this PR until this is resolved.